frontend/src/store.js

```diff
@@ -72,7 +76,7 @@ const store = createStore({
     async fetchRoom({ commit }, name) {
       let res = await fetch(`${backend}/api/room?name=${name}`, {
         method: 'get',
-        headers: { 'content-type': 'application/json' }
+        headers: { 'content-type': 'application/json' },
```

Comment:
Same here. You may have different local linter settings if this change weas done by vscode